app/controllers/service_controller.rb

```diff
@@ -42,11 +42,11 @@ def show
     @lastaction = "show"
 
     @gtl_url = "/show"
-    @display = params[:display] if params[:display]
+
+    set_display
```

Comment:
@AparnaKarve, @dclarizio : don't do/merge such things, please.

Problems with this:
 1. the show methods have certain structure. I and other people are working to refactor them all in the same way. If we refactor one of them in a different way, the situation is not getting any better. On the contrary. Anyone grepping the sources for patterns will miss this one. Also the automated tools looking for similar code will miss this one.
 1. This change replaces one issue with another. The 2 extracted lines are moved into method named `set_display`. That immediately creates an issue because methods should not be named `set_*` or `get_*`.

Thank you!

Reply:
The change was done to address cyclomatic complexity in the method.

Usually Rubocop explicitly mentions about `set_` or `get_` usage in method names -- but as you can see, in this case, Rubocop is all clean.
Has the `set_` /`get_` method name restriction been removed? -- we are not getting those warnings anymore.

Reply:
> The change was done to address cyclomatic complexity in the method.

I can see that in the commit message. However I don't find that information relevant in the context of my comment.

> Usually Rubocop explicitly mentions about set_ or get_ usage in method names -- but as you can see, in this case, Rubocop is all clean.

I am not aware of such change.

Reply:
@martinpovolny can you recommend the right way to fix this so we can change it upstream with a follow up PR?  Thx, Dan

Reply:
For this particular case, where the choice is between keeping an existing pattern that exists in the other controllers Vs fixing the cyclomatic complexity, I would favor fixing the cyclomatic complexity.

- This is a new controller. So why not keep things clean here?
- I think it is a good idea to have a separate method that sets `@display`, which has a special significance in the code, rather than blending it up with rest of the `show` method

Maybe the other older controllers should be having a dedicated method for setting `@display` too.

Reply:
> For this particular case, where the choice is between keeping an existing pattern that exists in the other controllers Vs fixing the cyclomatic complexity, I would favor fixing the cyclomatic complexity.

I would not.

It all boils down to a simple fact: If we refactor similar stuff differently in different places, we increase the cost of maintenance from that point in time until a point where someone actually unifies the similar places back.

I hope that it's clear that maintaining 2 different routines that do the same thing is more expensive than maintaining one.

Also the cost of refactoring that piece increases. Someone will have to find the places (that look less similar now), reason about why these are different: Is there some functional difference? Is there a bug? Then probably unify the code back and then replace it with refactored code.

I can try to reword the reasons behind that or I can show you a number of examples.

If you compare the cost of the above vs the cost of keeping the "cyclomatic complexity" warning in place would you still prefer "fixing" the problem?

I urge everyone to think in the context of the whole app. Not in the context of a single method or controller. If the controllers or methods in them that do the very same thing differ too much due to people refactoring each piece in a different way the project will not be maintainable at all.

Reply:
Ok, noted the above.

To remove this cyclomatic complexity fix in upstream, we could simply revert https://github.com/ManageIQ/manageiq-ui-classic/pull/2992/commits/644ad84e2b188526636e4b2543cb08208595634d